snowdrop-block/src/Snowdrop/Block/Application.hs

```diff
@@ -229,7 +229,8 @@ applyFork :: forall blkType m conf txtypes xs
     -> m (OldestFirst [] (ChgAccum conf))
 applyFork cfg fork ForkApplyAction{..} = do
     toApply <- txsToApply -- Apply tx1, ..., tx_k to state
-    forkUndos <- liftERoComp $ traverse (computeUndo @xs @conf) chgAccums
+    forkUndoss <- liftERoComp $ traverse (traverse (computeUndo @xs @conf)) chgAccums
```

Comment:
The code below makes little sense to me, we rollback blocks which should be applied before even applying them. This will only produce an error.


Instead, blunds following `faaRollbackTo` in chain shall be loaded and undo from these blunds applied

Reply:
> blunds following faaRollbackTo in chain shall be loaded and undo from these blunds applied

Is done as part of `processFork` ie. step 3.

I've removed `modifyAccumUndo` call and now we only use `computeUndo` to get undos to store in `Blund`s.